mixins/test/localize-mixin.html

```diff
@@ -61,74 +112,41 @@
 					});
 				});
 
-				describe('localize', () => {
-					let elem;
-					beforeEach(async() => {
-						elem = await fixtureAndAwaitRender('basic');
-					});
-					it('should localize text using object format', () => {
-						const val = elem.localize('hello', {name: 'Bill'});
-						expect(val).to.equal('Hello Bill');
-					});
-					it('should localize text using legacy format', () => {
-						const val = elem.localize('hello', 'name', 'Bill');
-						expect(val).to.equal('Hello Bill');
-					});
-					it('should re-localize text when locale changes', (done) => {
-						const valInitial = elem.localize('hello', {name: 'Sam'});
-						expect(valInitial).to.equal('Hello Sam');
-						const myEventListener = () => {
-							const val = elem.localize('hello', {name: 'Mary'});
-							expect(val).to.equal('Bonjour Mary');
-							elem.removeEventListener('d2l-localize-behavior-language-changed', myEventListener);
-							done();
-						};
-						elem.addEventListener('d2l-localize-behavior-language-changed', myEventListener);
-						documentLocaleSettings.language = 'fr';
-					});
-					it('should localize term using plurals', () => {
-						const val = elem.localize('plural', {itemCount: 1});
-						expect(val).to.equal('You have 1 item.');
-					});
-				});
 				describe('shouldUpdate tracking', () => {
 
 					it('should pass all changed properties to updated()', (done) => {
-						const elem = fixture('async');
+						const container = fixture('container');
```

Comment:
Needed to do a bit of hackery here and not use `fixture` -- by creating the component ourselves we can register for events that would normally fire before our fixture gets called.